docs/source/scope_tutorial.ipynb

```diff
@@ -0,0 +1,215 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "==============\n",
+    "Scope Analysis\n",
+    "==============\n",
+    "Scope analysis keeps track of assignments and accesses which could be useful for code automatic refactoring. If you're not familiar with Scope analysis, see :doc:`Metadata <metadata>` for more detail about Scope metadata. This tutorial demonstrates some use cases of Scope analysis. \n",
```

Comment:
> If you're not familiar with Scope analysis, see :doc:`Metadata <metadata>` for more detail about Scope metadata.

- Can you link to the "Scope Metadata" section of that page? Otherwise it's not clear what part of that page I should be looking at.
- Scope isn't a proper noun, so unless it's a reference to a class name or the start of a sentence, it shouldn't be capitalized.
- You might also want to link to the other metadata tutorial.